editor/find_in_files.cpp

```diff
@@ -221,7 +221,7 @@ float FindInFiles::get_progress() const {
 
 void FindInFiles::_scan_dir(String path, PoolStringArray &out_folders) {
 
-	DirAccess *dir = DirAccess::open(path);
+	DirAccessRef *dir = DirAccessRef::open(path);
```

Comment:
`DirAccessRef` is a wrapper to a pointer, there is no need to use a pointer type here